internal/controlplane/handlers_authz_test.go

```diff
@@ -157,6 +157,7 @@ func TestEntityContextProjectInterceptor(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
+		tc := tc
```

Comment:
Same comment as below - curious if you ran into a problem that required this to be reinserted.